### ISSUE TITLE
After applying high contrast black theme, visual focus indicator is not properly visible on toggle button group

### ISSUE THREAD

**Issue by yangster23** [react-bootstrap/react-bootstrap#5386]
After applying high contrast black theme on a toggle button group component, the visual focus indicator should be visible whenever one of the buttons is selected. This allows keyboard user to navigate properly on the controls. 

Using high contrast from Windows setting.

Violates MAS 4.3.1 – No Disruption of Accessibility Features. Same issue occurs on React bootstrap documentation - https://react-bootstrap.github.io/components/buttons/

![image](https://user-images.githubusercontent.com/5368099/90449808-a5d1e780-e094-11ea-9f09-138215b6607a.png)

**Comment by artcg** [comment]
That is interesting, but it looks like it is actually a problem with the browser / theme that you are using

The focus indicator works by adding a  box-shadow which your browser / theme seems to be electing not to display

In any case, this would not be an issue to raise with react-bootstrap but rather with [bootstrap](https://github.com/twbs/bootstrap) since it is a CSS issue and this library doesn't provide CSS
